install.sh

```diff
@@ -34,6 +45,22 @@ print_help()
    echo "   GATEWAY_IP: the public IP that will be used by LIQO to establish the interconnection with other clusters"
 }
 
+function wait_and_approve_csr(){
+   max_retry=3
+   retry=0
+   while [ "$retry" -lt "$max_retry" ]; do
+     set +e
```

Comment:
Is this necessary? It seems to me that only `kubectl certificate approve` may fail and you terminate in that case

Reply:
The problem is kubectl get csr which may fail and should not trigger an exit.